src/chi/components/tooltip/tooltip.scss

```diff
@@ -14,19 +16,35 @@
     font-size: $text;
     font-weight: normal;
     line-height: $line-height-small;
+    margin-right: -1rem;
+    max-height: $line-height-small * $multiLineEllipsisMax + 0.5rem;
     max-width: 20rem;
     opacity: 0;
     overflow: hidden;
-    padding: .25rem .5rem;
+    padding: .25rem 1rem;
     pointer-events: none;
-    text-overflow: ellipsis;
+    position: relative;
+    text-align: justify;
```

Comment:
@dani-cl-madrid need to achieve this without `text-align: justify;` as it impairs readability

Reply:
Justify was a must in the implemented solution, so I solved it via JavaScript. It uses dihcotomic search in order not to test al possible lengths of the string.